app/routes/downloadPaymentsByYearSummary.js

```diff
@@ -0,0 +1,32 @@
+const { Parser } = require('json2csv')
+const { getSchemePaymentsByYear } = require('../services/databaseService')
+
+module.exports = {
+  method: 'GET',
+  path: '/downloadPaymentsByYearSummary',
+  handler: async (_request, _response) => {
+    const fields = [
+      'financial_year',
```

Comment:
Sorry I missed this in my previous review, the field here will appear on the csv as column names so could we get rid of the underscore and perhaps capitalise the words like: Financial year, Scheme and Amount? This is not explicitly mentioned in the ticket but it would provide a better user experience.

Reply:
No Problem, I just update it.